make/e2e-setup.mk

```diff
@@ -145,7 +145,7 @@ $(call image-tar,kyverno) $(call image-tar,kyvernopre) $(call image-tar,bind) $(
 
 # Same as above, except it supports multiarch images.
 $(call image-tar,kind): bin/downloaded/containers/$(CRI_ARCH)/%.tar: bin/tools/crane
-	@$(eval IMAGE=$(subst +,:,$(shell cut -d/ -f2- <<<$*)))
```

Comment:
I don't know why this was needed, but it resulted in incorrect image name for 1.24 image as the image name format is different.
Again, IMO it would be awesome if we could find some higher level tools to do these kind of actions and also move them into separate scripts.

Reply:
I copied this from the target above [here](https://github.com/cert-manager/cert-manager/blob/master/make/e2e-setup.mk#L139).
As I understand from [this comment](https://github.com/cert-manager/cert-manager/blob/master/make/e2e-setup.mk#L91-L108) `image-tar kind` [here](https://github.com/cert-manager/cert-manager/blob/master/make/e2e-setup.mk#L147) swaps potential `:`s in the tarfile name for `+`s to match makefile name formatting and the subst then swaps them back I guess. Perhaps we could look into doing tar-ing without this in another PR.